File: C++/variables.cpp
#include <iostream>
#include <string>

int main() {

    int age = 25;
    float height = 1.75f;
    double weight = 70.5;
    char initial = 'A';
    bool isStudent = true;

    std::string name = "John Doe";
    std::cout << "Name: " << name << std::endl;
    std::cout << "Age: " << age << std::endl;
    std::cout << "Height: " << height << " meters" << std::endl;
    std::cout << "Weight: " << weight << " kg" << std::endl;
    std::cout << "Initial: " << initial << std::endl;
    std::cout << "Is Student: " << (isStudent ? "Yes" : "No") << std::endl;

    return 0;
}

File: JS/variables.js
let a = 10;
const b = 11;

console.log(a,b)

let user = "Jane";
let greeting = `Hello, ${user}! How are you today?`;

let x = [1,2,3,4,5]

const person1 = {
    firstName: "John",
    lastName: "Doe",
    age: 30,
    greet: function() {
      console.log(`Hello, my name is ${this.firstName} ${this.lastName}!`);
    }
  };

// Spread Operator : 
let parts = ['shoulders', 'knees'];
let lyrics = ['head', ...parts, 'and', 'toes']; 

let person = { name: 'John', age: 25 };
let location = { country: 'USA', city: 'San Francisco' };
let profile = { ...person, ...location };
